cli/replicasets/bootstrap_vshard.go

```diff
@@ -20,7 +20,7 @@ func BootstrapVshard(ctx *context.Ctx, args []string) error {
 		return fmt.Errorf("failed to bootstrap vshard: %s", err)
 	}
 
-	log.Infof("Vshard is bootstrapped successfully")
+	log.Infof("Bootstrap vshard task was succesful, check your cluster status")
```

Comment:
To be honest, I don't get why this message should cover the case "vshard was bootstrapped with partial success and some errors". Or it isn't?

Reply:
It's meaning is that cartridge-cli function worked succesfully (sent request to cartridge), but it is'n checking for succes of bootstrapping itself, so user can check cluster status.  More about why we can not check was vshard bootstrapped or not i wrote in the Pull request body.